stonesoup/resampler/particle.py

```diff
@@ -24,6 +25,7 @@ def resample(self, particles):
 
         n_particles = len(particles)
         weight = Probability(1/n_particles)
+        particles = sorted(particles, key=attrgetter('weight'), reverse=False)
         cdf = np.cumsum([p.weight for p in particles])
         particles_listed = list(particles)
```

Comment:
Just realised that as `particles` is sorted on line 28 (R), that it will already be a list.

Reply:
Well spotted 👍 That's been corrected now.